src/control/server/init/setup_spdk.sh

```diff
@@ -10,6 +10,55 @@ set -e
 rootdir="$(readlink -f "$(dirname "$0")")"/../..
 scriptpath="$rootdir/spdk/scripts/setup.sh"
 
+#
+# Sets appropriate permissions on /dev/vfio/* files
+#
+set_vfio_permissions()
+{
+	# make sure regular users can read /dev/vfio
+	echo "chmod /dev/vfio"
+	sudo chmod a+x /dev/vfio
+	if [ $? -ne 0 ] ; then
+		echo "FAIL"
+		return
+	fi
+	echo "OK"
+
+	# make sure regular user can access everything inside /dev/vfio
+	echo "chmod /dev/vfio/*"
+	sudo chmod 0666 /dev/vfio/*
+	if [ $? -ne 0 ] ; then
+		echo "FAIL"
+		return
+	fi
+	echo "OK"
```

Comment:
I'm not sure this is correct from a security perspective. There must be a reason that the default permissions don't allow access by regular users. Seems like it would be safer to grant limited permissions to the user running daos_io_server.

Reply:
I'm afraid that's what they do in the dpdk-script.sh

Reply:
Hmm. All the more reason to strongly considering removing our dependency on these external scripts.